spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,110 @@
+require "spec_helper"
+
+describe "FacilityFactory" do
+  before(:each) do
+    @facility_factory = FacilityFactory.new
+    @or_dmvs = DmvDataService.new.or_dmv_office_locations
+    @ny_dmvs = DmvDataService.new.ny_dmv_office_locations
+    @mo_dmvs = DmvDataService.new.mo_dmv_office_locations
+  end
+
+  describe "#initialize" do
+    it "exists" do
+      expect(@facility_factory).to be_a(FacilityFactory)
+    end
+  end
+
+  describe "#create_facilities" do
+    it "returns an array of Facilities" do
+      facilities = @facility_factory.create_facilities(@or_dmvs)
+      first_facility = facilities.first
+      
+      expect(facilities).to be_a(Array)
+      expect(facilities.count).to eq(59)
+      expect(first_facility).to be_a(Facility)
+    end
+  end
+
+  describe "#format_name" do
```

Comment:
Great job testing your helper methods!!! For these ones though, I'd actually want to test actual data. You could input your own data to ensure everything is being formatted correctly, so you could set or_facility to something specific, a hash that could come back from the api, and then test that your `format_name` method actually formats it the way you would expect. Does that make sense? If not, let me know, happy to explain further.

Reply:
Same with the helper methods below. I really really appreciate that you tested them. And in the future, if you know what the input to the method will be, you can create that fake data on your own (like how we setup a fake Facility object to test all the Facility methods) and ensure that your methods gives the correct output based on the fake input.